hack/lint.sh

```diff
@@ -4,7 +4,7 @@ RETVAL=0
 GENERATED_FILES="zz_generated.*.go"
 for file in $(find . -path ./vendor -prune -o -type f -name '*.go' -print | grep -E -v "$GENERATED_FILES"); do
 	golint -set_exit_status "$file"
-	if [[ $? -eq 1 ]]; then
+	if [[ $? -ne 0 ]]; then
```

Comment:
Just a curiosity: Why this? Does golint return something other than 0 or 1?

Reply:
If so, please include a reasoning in the commit message. Just being thorough. :)

Reply:
if golint binary doesn't exist, it's a different error code than 1, which made it look like lint was passing in my 'tools' container.

in general, these sorts of tests should be checking for non-zero exit codes to avoid silently passing for unexpected reasons like this.

Reply:
Good enough!